src/lib/third_party/include/gcrypt_light.h

```diff
@@ -18,7 +18,7 @@
 #define GPG_ERR_ANY -2
 #define GPG_ERR_INV_ARG -3
 #define GCRYCTL_INITIALIZATION_FINISHED_P 1,0
-#define GCRYCTL_INITIALIZATION_FINISHED 2
+#define GCRYCTL_INITIALIZATION_FINISHED 1
```

Comment:
Why change it?

Reply:
`gcry_control()` and the defines `GCRYCTL_INITIALIZATION_FINISHED*` are useless when using the internal crypto code, but they are needed to be compatible con external libgcrypt.
With internal crypto, `gcry_control()`  must return true.
I reworked the code to make it clearer